services/devices/cli/app.py

```diff
@@ -0,0 +1,147 @@
+
+"""
+This CLI app is responsible for handling the actions to control the worker app.
+It create/update/delete sqs message command to control the worker app.
+Once worker app receive the sqs message, it will handle the action.
+"""
+
+
+import os
+import uuid
+from enum import Enum
+import pandas as pd
+import random
+import click
+
+from models_and_classes.create_agents import generate_first_number_agents_from_simulation_csv_file, Market_Interval, Device_Type, generate_emulated_battery_csv_file_with_device_id, BATTERY_BRANDS, ECS_ACTIONS_ENUM
+from models_and_classes.destroy_agents import destroy_all
+import logging
+logging.basicConfig(format='%(asctime)s %(message)s',
+                    datefmt='%m/%d/%Y %I:%M:%S %p', level=logging.INFO)
+
+logging.info(f"Start the worker app")
+
+""""
+This controller app is used to control the worker app for developement and testing purpose.
+It will generate following sqs:
+1. generate number agents for test purpose
+2. destroy number agents
+
+"""
+
+SQS_GROUPID = os.environ['SQS_GROUPID']
```

Comment:
You should use "os.environment.get('SQS_GROUPID')" here, if the key is not set for some reason, it will throw a keyError exception.